po/installation/ar.po

```diff
@@ -66,18 +66,18 @@ msgstr "منافذ VNC ستكون مفتوحة (<a href=\"%s\">حظر</a>)"
 
 #: src/lib/y2firewall/clients/proposal.rb:133
 msgid "VNC ports will be blocked (<a href=\"%s\">open</a>)"
-msgstr "VNC سيكون مغلق (<a href=\"%s\">فتح</a>)"
+msgstr "‏VNC سيكون مغلق (<a href=\"%s\">فتح</a>)"
 
 #. Returns the SSH-port part of the firewall proposal description
 #. Returns nil if this part should be skipped
 #. @return [String] proposal html text
 #: src/lib/y2firewall/clients/proposal.rb:144
 msgid "SSH port will be open (<a href=\"%s\">block</a>)"
-msgstr "SSH سيتم فتحه (<a href=\"%s\">أغلق</a>)"
+msgstr "‏SSH سيتم فتحه (<a href=\"%s\">أغلق</a>)"
```

Comment:
The GitHub rendered diff does not show any difference, but the actual change is adding a Right-To-Left mark (U+200F) at the beginning of the string, before "SSH" or "VNC" can establish a LTR direction.